dynamo/vectorfield/clustering.py

```diff
@@ -196,18 +206,21 @@ def cluster_field(
                 adata,
                 adj_matrix_key="vf_feature_knn",
                 result_key="field_leiden",
+                **kwargs
             )
         elif method == "louvain":
             louvain(
                 adata,
                 adj_matrix_key="vf_feature_knn",
                 result_key="field_louvain",
+                **kwargs
             )
         elif method == "infomap":
             infomap(
                 adata,
                 adj_matrix_key="vf_feature_knn",
                 result_key="field_infomap",
+                **kwargs
```

Comment:
infomap also seem to don't have a resolution parameter: https://cdlib.readthedocs.io/en/latest/reference/cd_algorithms/algs/cdlib.algorithms.infomap.html

Reply:
btw, leiden also doesn't seem to have the resolution parameter (we used cdlib): https://cdlib.readthedocs.io/en/latest/reference/cd_algorithms/algs/cdlib.algorithms.leiden.html?highlight=leiden

Reply:
As for leiden, 
leiden should have a resolution parameter accroding to their code https://github.com/vtraag/leidenalg/blob/91fbe8ced00f7e0905578583d8b748388dc84b1b/src/leidenalg/functions.py#L30

scanpy's leiden has a resolution parameter (https://scanpy.readthedocs.io/en/latest/generated/scanpy.tl.leiden.html)
https://github.com/scverse/scanpy/blob/bd06cc3d1e0bd990f6994e54414512fa0b25fea0/scanpy/tools/_leiden.py#L142

I think cdlib's developer has not been aware of the feasibility of setting resolutions. 

It would be a better idea to write a leiden wrapper like scanpy than using cdlib. After all, for clustering analysis, resolution matters.

Reply:
As for infomap,

there is no direct "resolution parameter" in their docs (https://mapequation.github.io/infomap/python/infomap.html).

but there are two parameters having impact on the clustering granularities: preferred_number_of_modules, depth_level.

though cdlib doesn't directly provide a parameter altering clustering granularities(https://cdlib.readthedocs.io/en/latest/reference/cd_algorithms/algs/cdlib.algorithms.infomap.html), I guess the two params i mentioned ( preferred_number_of_modules, depth_level) can be passing via the `flag`. 

My personal opinion is cdlib is a good ensemble but still  on its way to maturity. sometimes it might be easier for developers to directly wrap up the original algorithm than using them as a proxy.